magda-registry-api/src/main/scala/au/csiro/data61/magda/registry/RecordPersistence.scala

```diff
@@ -393,7 +394,11 @@ object DefaultRecordPersistence
           from RecordAspects
           inner join Aspects using (aspectId, tenantId)
           inner join Records using (recordId, tenantId)
-          where (RecordAspects.recordId, RecordAspects.aspectId, RecordAspects.tenantId)=($recordId, $aspectId, $tenantId)
+where (RecordAspects.recordId, RecordAspects.aspectId)=($recordId, $aspectId) AND ${SQLUtil
+      .tenantIdToWhereClause(
+        tenantId,
+        Some(SQLSyntax.createUnsafely("RecordAspects"))
```

Comment:
Why pass `SQLSyntax.createUnsafely("RecordAspects")` instead of "RecordAspects"?
In `tenantIdToWhereClause`, It seems we pass the content through `SQLSyntax.createUnsafely` as well?

Reply:
There's a bit of explanation in the scaladoc - essentially because if it just accepted a String, the person calling the function might not realise that the content of that string is going to be passed unsafely into SQL. By requiring an SQL statement, they have to use `createUnsafely` which means they'll be aware to make sure user input doesn't get passed in (hopefully).

The other alternative is to accept an enum or something, but then the enum needs to be updated whenever a table name changes.

Reply:
👍